src/main/java/kata/makers/service/FlightService.java

```diff
@@ -0,0 +1,56 @@
+package kata.makers.service;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Scanner;
+import kata.makers.exception.CreateFlightException;
+import kata.makers.model.Flight;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class FlightService {
+  private Map<String, Flight> flightInformation;
+  private static final Logger log = LoggerFactory.getLogger(FlightService.class);
+  public FlightService(Map<String, Flight> flightInformation){
+   this.flightInformation = flightInformation;
+  }
+
+  public void listAll() {
+    if (flightInformation.isEmpty()) {
+      log.info("no flight here");
+    } else {
+      for (Map.Entry<String, Flight> entry : flightInformation.entrySet()) {
+        log.info("Flight number {} - {}", entry.getKey(), entry.getValue());
+      }
+    }
+  }
+
+  public Optional<Flight> getFlightByNumber(String flightNumber){
+    return Optional.ofNullable(flightInformation.get(flightNumber));
+  }
+
+  public void create() throws CreateFlightException{
```

Comment:
Maybe `createFlight()` would be a better name? It's a flightService method so it may be obvious, but then we have `createReservation() `etc in other places, so being more specific could be better